packages/core/src/components/block/block.tsx

```diff
@@ -14,6 +19,19 @@ export class TdsBlock {
   /** Mode variant of the component, based on current mode. */
   @Prop() modeVariant: 'primary' | 'secondary' = null;
 
+  /** Specifies the HTML tag to be used for the component wrapper.
+   *
+   *  @default 'div'. */
```

Comment:
```suggestion
  /** Specifies the HTML tag to be used for the component wrapper. */
```
Default is automatically set based on code below, no need for extra highlight. 
If this is something we would like to keep using, lets discuss in devCheckin, otherwise lets keep code standardized.